### ISSUE TITLE
Running locally seems to be "broken"

### ISSUE THREAD

**Issue by karfau** [github/super-linter#212]
**Describe the bug**
when running the super-linter locally as described [here]() it reports a success but it didn't lint anything.

**To Reproduce**
```
$ mkdir __test__
$ docker run -e RUN_LOCAL=true -v __test__:/tmp/lint github/super-linter:v2.1.0
--------------------------------------------------------------------------------
[... nice ASCI ART truncated ...]
--------------------------------------------------------------------------------

---------------------------------------------
--- GitHub Actions Multi Language Linter ----
---------------------------------------------

---------------------------------------------
The Super-Linter source code can be found at:
 - https://github.com/github/super-linter
---------------------------------------------
--------------------------------------------
Gathering GitHub information...
NOTE: ENV VAR [RUN_LOCAL] has been set to:[true]
bypassing GitHub Actions variables...
Linting all files in mapped directory:[/tmp/lint]

--------------------------------------------
Gathering user validation information...
awk: cmd. line:1: Unexpected end of string
awk: cmd. line:1: Unexpected end of string

----------------------------------------------
----------------------------------------------
The script has completed
----------------------------------------------
----------------------------------------------


All file(s) linted successfully with no errors detected
----------------------------------------------
```

**Expected behavior**
- I expect the step "Gathering user validation information` shouldn't fail locally. (Should it even be executed?)
- If it does the container should either exit or at least report that something didn't work.

**Comment by Hanse00** [comment]
Hey @karfau! The `awk` error messages you were seeing were fixed in version v2.1.1.

Can you try again with that version, and let us know if you’re still seeing any issues?

Thanks!

**Comment by karfau** [comment]
Well yes, but even when mapping my typescript source directory there is still no indication that it linted any file.
Would it be a good idea to either indicate how many files it found or even change the message to "No files found for linting" when this is the case?

**Comment by r35krag0th** [comment]
> 
> 
> Hey @karfau! The `awk` error messages you were seeing were fixed in version v2.1.1.
> 
> Can you try again with that version, and let us know if you’re still seeing any issues?
> 
> Thanks!

I don't see a release for 2.1.1.

**Comment by admiralAwkbar** [comment]
The release has been published and newer versions available without bug :)
